android/build.gradle

```diff
@@ -229,7 +229,9 @@ def JS_RUNTIME = {
         return "v8"
     }
     def appProject = findProject(":app") ? project(":app") : null
-    return appProject?.ext?.react?.enableHermes ? "hermes" : defaultRuntimeType
+    return "hermes"
+    // TODO: detect if Hermes is enabled
+    // return appProject?.ext?.react?.enableHermes ? "hermes" : defaultRuntimeType
```

Comment:
You should be able to either:
1. Read `appProject?.hermesEnabled`. If null fallback to 
2. `appProject?.ext?.react?.enableHermes`. If null fallback to
3. defaultRuntimeType